cucumber-expressions/java/src/main/java/io/cucumber/cucumberexpressions/BuiltInParameterTransformer.java

```diff
@@ -43,7 +43,7 @@ private Object doTransform(String fromValue, Type toValueType, Type originalToVa
         }
 
         if ((Character.class.equals(toValueClass) || char.class.equals(toValueClass)) && fromValue.length() == 1) {
-                return fromValue.toCharArray()[0];
+                return fromValue.charAt(0);
```

Comment:
Little bit of a nit-pick but I initially misread this condition. Might be good to put the `fromValue.length() == 1` in a seperate if statement.

Reply:
Sure...